### DISCUSSION TITLE
gdal and libkml

### DISCUSSION THREAD

**Discussion by gregseb** [Homebrew/discussions#4407] [Everyday usage] [answered]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.0.11
ORIGIN: https://github.com/Homebrew/brew
HEAD: 931327df1f30a3a5b1aa07657c0d1070e61988f9
Last commit: 11 days ago
Core tap origin: https://github.com/Homebrew/homebrew-core
Core tap HEAD: d49174a49f683f8849bde847db5dab8bd104baa5
Core tap last commit: 8 weeks ago
Core tap branch: master
Core tap JSON: 04 Apr 16:49 UTC
HOMEBREW_PREFIX: /opt/homebrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_EDITOR: vi
HOMEBREW_MAKE_JOBS: 8
Homebrew Ruby: 2.6.10 => /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/bin/ruby
CPU: octa-core 64-bit arm_firestorm_icestorm
Clang: 14.0.0 build 1400
Git: 2.37.1 => /Library/Developer/CommandLineTools/usr/bin/git
Curl: 7.86.0 => /usr/bin/curl
macOS: 13.2.1-arm64
CLT: 14.2.0.0.1.1668646533
Xcode: N/A
Rosetta 2: false
```

### Output of `brew doctor`

```shell
Your system is ready to brew.
```

### Description of issue

Is there any reason to not add libkml support to the gdal formula? I've tested locally and the only thing that needs changed is for `depends_on "libkml"` to be added to gdal.rb

The libkml driver allows gdal to more correctly parse advanced features from kml files than the builtin xml parser.

Also the debian and ubuntu package `gdal-bin` both come with libkml enabled by default which has been causing us some difficulties when building/testing code that depends on gdal on our workstations.

**Comment by ZhongRuoyu** [comment, marked as answer]
That sounds reasonable. Let's do it in https://github.com/Homebrew/homebrew-core/pull/128083.

**Reply by ZhongRuoyu** [reply]
https://github.com/Homebrew/homebrew-core/pull/128083 is merged now. Please try:

```
brew update
brew reinstall gdal
```

**Reply by gregseb** [reply]
Wow awesome, fast work! Our team is thrilled! 💯
